v2/api-validator/src/config/index.ts

```diff
@@ -43,6 +43,36 @@ const config = convict({
     default: 2,
     env: 'PAGINATION_LIMIT',
   },
+  collateral: {
+    signers: {
+      userId: {
+        format: Array<string>,
+        default: ['be5ff76b-6cc8-4a39-af40-fc71aea1865c', 'c5bb0880-523c-4c1c-bfe7-bcf2941517a0'],
+        env: 'COLLATERAL_SIGNERS',
+      },
+    },
+    collateralAccount: {
+      accountId: {
+        format: String,
+        default:
+          'c8dd0660-555c-4c1c-bfe7-bcf2941517a0.be5ff76b-6cc8-4a39-af40-fc71aea1865c.c5bb0880-523c-4c1c-bfe7-bcf2941517a0',
+        env: 'COLLATERAL_ACCOUNT',
+      },
+    },
+    withdrawal: {
+      addresses: {
+        format: Array<PublicBlockchainAddress>,
+        default: [
+          {
+            address: 'J4NOFD4VBNJ35F2MEII4HRAADNPJ7QFYAKESYKSEWWGJUXG64IATUVZRMQ',
+            addressTag: '5494396EB65E6615D307',
+          },
+          { address: 'J4NOFD4VBNJ35F2MEII4HRAADNPJ7QFYAKESYKSEWWGJUXG64IATUVZRMQ', addressTag: '' },
+        ],
```

Comment:
These are not valid instances of `PublicBlockchainAddress` — `asset` and `transferMethod` properties are required.

Reply:
updated scheme and README.MD